pkg/controller/storagecluster/update.go

```diff
@@ -124,6 +144,38 @@ func (c *Controller) rollingUpdate(cluster *corev1.StorageCluster, hash string)
 	return c.syncNodes(cluster, oldPodsToDelete, []string{}, hash)
 }
 
+func getPxAPiPodForNode(nodeName string, ApiPods []*v1.Pod) *v1.Pod {
+
+	for _, pod := range ApiPods {
+		fmt.Println("PXPOD 2", pod.Spec.NodeName)
```

Comment:
please convert to `logrus.Debugf()` or `logrus.Tracef()` if not required to always log this